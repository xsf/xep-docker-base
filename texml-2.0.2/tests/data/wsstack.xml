<TeXML>

	[spaces eliminated]

	<TeXML ws="1">
		[ws around]
		<TeXML ws="1">
			[more ws around]
			<TeXML>
				[ws still around]
				<TeXML ws="0">
					[and now no ws]
					<TeXML ws="1"><TeXML>[again ws]
					</TeXML></TeXML>
				</TeXML>
			</TeXML>
		</TeXML>
	</TeXML>

	
</TeXML>
