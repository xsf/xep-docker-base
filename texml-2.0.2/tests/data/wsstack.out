[spaces eliminated]
		[ws around]
		%
			[more ws around]
			%
				[ws still around]
				 [and now no ws] [again ws]
					%
			%
		%
	%
